package.json

```diff
@@ -31,6 +31,7 @@
     "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
+    "lint-staged": "lint-staged",
```

Comment:
This wouldn't be needed if we change to `pnpm exec`. Yarn didn't differentiate between running a script and executing a binary from an installed package. At least yarn v1. PNPM (and NPM) have different commands for it: `pnpm run` to execute a script and `pnpm exec` to execute a binary of an installed dependency.

Reply:
That's awesome! I did not know about `pnpm exec`. I've updated it.